cves/CVE-2016-2162.yml

```diff
@@ -103,9 +109,11 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: |
+    No tests were changed in between and including in the VCC and the fix commit
```

Comment:
I18NInterceptor was unit tested both prior to and after the fix (see https://github.com/apache/struts/blob/STRUTS_2_3_28_1/xwork-core/src/test/java/com/opensymphony/xwork2/interceptor/I18nInterceptorTest.java).  However, it does not appear that they added another test for this issue specifically--only modified a few tests.